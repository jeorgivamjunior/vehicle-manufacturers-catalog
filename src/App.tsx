import { FC } from 'react';

import { Container, CssBaseline } from '@mui/material';

import { Header } from './components';
import { Manufacturer } from './pages';

import '@fontsource/roboto/400.css';

export const App: FC = () => {
  return (
    <>
      <Header />
      <Container component="main" maxWidth="sm">
        <CssBaseline />
        <Manufacturer />
      </Container>
    </>
  );
};
