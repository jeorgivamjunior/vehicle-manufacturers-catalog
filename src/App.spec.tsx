import renderer from 'react-test-renderer';

import {App} from './App';

it.skip('renders correctly', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
