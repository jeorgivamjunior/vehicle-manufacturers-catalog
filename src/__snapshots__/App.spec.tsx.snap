// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <header
    className="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionRelative css-1d22kfw-MuiPaper-root-MuiAppBar-root"
  >
    <div
      className="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-12oimmw-MuiToolbar-root"
    >
      <img
        className="css-z6u0ty"
        src="car.png"
      />
      <h4
        className="MuiTypography-root MuiTypography-h4 css-aziax4-MuiTypography-root"
      >
        Vehicle Manufacturers Catalog
      </h4>
    </div>
  </header>,
  <main
    className="MuiContainer-root MuiContainer-maxWidthMd css-xdmu94-MuiContainer-root"
  >
    <section
      className="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
    >
      <div
        className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation3 css-3su884-MuiPaper-root"
      >
        <div
          className="css-1n168u8"
        >
          <div
            aria-colcount={4}
            aria-multiselectable={false}
            aria-rowcount={1}
            className="MuiDataGrid-root MuiDataGrid-root--densityStandard css-1e2bxag-MuiDataGrid-root"
            role="grid"
          >
            <div
              className="MuiDataGrid-main css-204u17-MuiDataGrid-main"
            >
              <div
                className="MuiDataGrid-overlay css-f7tavo-MuiDataGrid-overlay"
                error={[TypeError: Cannot set properties of null (setting 'scrollLeft')]}
              >
                An error occurred.
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
  </main>,
]
`;
